### PR TITLE
Add compatibility note to S3 file system page

### PR DESCRIPTION
**Pull request by michaeleby1**
## Description

Add same note as #22711 to S3 file system page, indicating that we do not test all S3-compatible storage systems

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/object-storage/file-system-s3.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 Trino includes a native implementation to access [Amazon
 S3](https://aws.amazon.com/s3/) and compatible storage systems with a catalog
-using the Delta Lake, Hive, Hudi, or Iceberg connectors.
+using the Delta Lake, Hive, Hudi, or Iceberg connectors. While the exchange
+manager is designed to support S3-compatible storage systems, not all such
```
Comment (mosabua) [inline docs/src/main/sphinx/object-storage/file-system-s3.md:6]: @electrum @findinpath @ebyhr .. do we want to clarify that we test with S3 and I think also MinIO?

**Review by mosabua — COMMENTED** [review]
Looks good in principal but I think we might want to elaborate some more, depending on the technical feedback.

File: docs/src/main/sphinx/object-storage/file-system-s3.md (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/object-storage/file-system-s3.md:6]: Sounds good .. can you address that @michaeleby1

File: docs/src/main/sphinx/object-storage/file-system-s3.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 Trino includes a native implementation to access [Amazon
 S3](https://aws.amazon.com/s3/) and compatible storage systems with a catalog
-using the Delta Lake, Hive, Hudi, or Iceberg connectors.
+using the Delta Lake, Hive, Hudi, or Iceberg connectors. While the exchange
```
Comment (electrum) [inline docs/src/main/sphinx/object-storage/file-system-s3.md:5]: This should not say "exchange manager"

**Comment by electrum** [comment]
I think we should remove this wording from the exchange manager documentation. It's very unlikely vendor documentation will tell the user about compatibility with Trino.

**Comment by mosabua** [comment]
> I think we should remove this wording from the exchange manager documentation. It's very unlikely vendor documentation will tell the user about compatibility with Trino.

Fair... maybe we should tell to check vendor docs and basically test themselves or something ..  how do we want to guide users around the "S3-compatibility" from various systems when we just test with s3 and minio .. and even there we just use it in our testing but as open source project obviously dont offer "support"

**Comment by electrum** [comment]
The user needs to test it themselves and contact the vendor with any questions.

**Comment by mosabua** [comment]
> The user needs to test it themselves and contact the vendor with any questions.

Ok .. can you reword it along those links here and in the exchange section @michaeleby1 .. and mention what Trino tests run with (S3 and minio) but there should be no perception of "support"

**Review by mosabua — CHANGES_REQUESTED** [review]
One nit .. and also port these adjustments to the exchange manager section.

**Review by mosabua — APPROVED** [review]
Looks good now. I will merge after release completed.

**Pull request merged**